modin/experimental/scripts/configure_ray_cluster.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/sh
```

Comment:
I've made some scripts like this in the past, and in the end I've always had to convert them from bash -> Python. Happy to share examples.

Reply:
Would love to see!